### ISSUE TITLE
Could you provide the training set and test set.

### ISSUE THREAD

**Issue by December26** [Leavingseason/dfm#1]
Many thanks!
